src/dom-to-image-more.js

```diff
@@ -887,18 +887,26 @@
                 });
 
             function inlineBackground(backgroundNode) {
-                const background = backgroundNode.style.getPropertyValue('background');
+                const properties = ['background', 'background-image'];
 
-                if (!background) { return Promise.resolve(backgroundNode); }
+                const inliningTasks = properties.map(function (propertyName) {
+                    const value = backgroundNode.style.getPropertyValue(propertyName);
 
-                return inliner.inlineAll(background)
-                    .then(function (inlined) {
-                        backgroundNode.style.setProperty(
-                            'background',
-                            inlined,
-                            background
-                        );
-                    })
+                    if(!value) {
+                        return Promise.resolve();
+                    }
+
+                    return inliner.inlineAll(value)
+                        .then(function (inlined) {
+                            backgroundNode.style.setProperty(
+                                propertyName,
+                                inlined,
+                                backgroundNode.style.getPropertyPriority(propertyName)
```

Comment:
this should be `value` that we computed on line 893

Reply:
The third argument for `CSSStyleDeclaration.setProperty()` is `priority`. The fact that we previously provided `background` (synonymous with `value` after my refactor) is the main cause of issue #72. If the `priority` argument contains anything other than `"important"`, `""`, or `undefined` then the `setProperty` call is a no-op. (Tested in both Firefox and Chrome.

I added support for `background-image` (and potentially other properties in future) as a bonus, but the priority fix is the main improvement of this PR.

I have updated the PR to make the code clearer.

Reply:
You're 100% correct, sorry. Will test this weekend